### ISSUE TITLE
RefTeX cannot find .bib file or cross references in multifile document

### ISSUE THREAD

**Issue by davidpasztor** [syl20bnr/spacemacs#10822]
#### Description :octocat:
I have a large LaTex project, which I organized into several `.tex` files and use `\input{}` in a single file to include the content of the other files. However, when editing this project from spacemacs, I run into some trouble, because spacemacs cannot find cross references from the other `.tex` files nor can it find citations from my `.bib` file at all.

Autocomplete only works for labels defined in the same `.tex` file when I use `\ref{}`, while autocomplete doesn't work at all when using `\cite{}`

#### Reproduction guide :beetle:
- Start Emacs
- Create a LaTeX project with several `.tex` files and try to cross reference them/use citations.

My project structure is the following: I have a `makefile` in the root directory in my project and all `.tex` files along with the `bib` file is located in a subdirectory. I include the bibliography in my main `.tex` file using `\bibliography{Texfiles/references.bib}`.

*Observed behaviour:* :eyes: :broken_heart:
when using `\cite{}`, I receive the following error when trying to use autocomplete in a `cite` command: 
>cite: info not available, use ‘M-RET r v’ to get it.

Then if I push `M-RET r v`, I receive a new error:
>reftex-view-cr-cite: Cannot display crossref

Calling `reftex-citation` results in the following error:
>reftex-offer-bib-menu: No valid bibliography in this document, and no default available

*Expected behaviour:* :heart: :smile:
Autocomplete works by listing all available citations from the `.bib` file specified in the `\bibliography` command when using the `\cite` command and lists all cross references from labels from all `.tex` files when using the `\ref` command.

#### System Info :computer:
- OS: darwin
- Emacs: 27.0.50
- Spacemacs: 0.300.0
- Spacemacs branch: develop (rev. 4980a646)
- Graphic display: t
- Distribution: spacemacs
- Editing style: vim
- Completion: helm
- Layers:
```elisp
(typescript latex bibtex helm auto-completion emacs-lisp git markdown org spell-checking syntax-checking themes-megapack)
```
- System configuration features: RSVG IMAGEMAGICK NOTIFY ACL GNUTLS LIBXML2 ZLIB TOOLKIT_SCROLL_BARS NS MODULES THREADS

**Comment by dbrw** [comment]
any work around for this, @davidpasztor ?

**Comment by inzouzouwetrust** [comment]
Bump!

**Comment by davidpasztor** [comment]
@inzouzouwetrust unfortunately I haven't used spacemacs for quite a while now, so not sure if this has been fixed since.

**Comment by joelstrouts** [comment]
I also have this problem

**Comment by syl20bnr** [comment]
I'm not familiar with LaTeX so it is a bit hard to handle this kind of bug.

Can someone provide some files so we can try to reproduce this bug ?

**Comment by dustiestgolf** [comment]
I found that I am able to replicate this bug without needing a large latex project or even alternate files accessed via `\include`. I have attached a [zip](https://github.com/syl20bnr/spacemacs/files/6308931/reftex_bug_example.zip) file with an example. The error doesn't always occur though. It seems to be happening for me on the initial citation. If I have citations already in the document when I load it I can sometimes get autocompletion help for the citations already in the document but no help on citations in the `.bib` file. 

Hope this helps!
